salesforcecli/plugin-deploy-retrieve
# Sh/deploy zip size

### What does this PR do?
Requires and consumes [SDR PR 1403](https://github.com/forcedotcom/source-deploy-retrieve/pull/1403); the zipSize and zipFileCount fields on SDR's DeployResult and the deployZipData event.  Displays this zip data in deploy command output and JSON response.

### What issues does this PR fix or reference?
@W-16488140@

To test:
link SDR PR 1403 branch to this branch
Run deploy commands of various kinds with and without `--json`.  Note the zip size and zip file count output and fields.
Turn on debug output and look for debug lines about zip size and file count
check out telemetry event with this data added
the JSON response with deploy commands will always have the zipSize and zipFileCount fields added, as long as the data is there in the event.
